### DISCUSSION TITLE
Cannot approve own revisions

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Revisions#1060] [Feature Requests]
If a user has edit_published capability, they will be able to approve any revisions.

It will be great if we can distinguish it and they can only approve the revisions that are not made by them.

It will be useful for the case when the user has 2 roles, they are an editor and also a revisor

**Comment by KratosGemini** [comment]
Maybe you are already thinking of this. My input on this would be to make sure users could still potentially approve their own revisions if we want to set it up that way. It's particularly helpful for site Administrators in case a developer makes a revision that needs to be released quickly, but no other developer is available to approve it at the moment.
